source/operations/concepts/erasure-coding.rst

```diff
@@ -10,62 +10,131 @@ Erasure Coding
+
+   If Parity (``EC:M``) is exactly 1/2 the number of drives in the erasure set, **write quorum** is ``K+1``.
+   This extra shard prevents data inconsistency due to "split-brain"-types of failure states
 
 .. _minio-ec-erasure-set:
 
-Erasure Sets
-------------
+Erasure Set and Distribution
+----------------------------
+
+An **Erasure Set** is a group of drives onto which MinIO can read or write erasure coded objects.
+   .. figure:: /images/erasure/erasure-coding-erasure-set.svg
+      :figwidth: 100%
+      :align: center
+      :alt: Diagram of erasure set covering 4 nodes and 16 drives
 
-An *Erasure Set* is a group of drives onto which MinIO writes erasure coded objects.
-MinIO randomly and uniformly distributes the data and parity shards of a given object across the erasure set drives, where a given drive has no more than one block of either type per object (no overlap).
- 
-MinIO automatically calculates the number and size of Erasure Sets ("stripe size") based on the total number of nodes and drives in the :ref:`Server Pool <minio-intro-server-pool>`, where the minimum stripe size is 2 and the maximum stripe size is 16.
-All erasure sets in a given pool have the same stripe size, and MinIO never modifies nor allows modification of stripe size after initial configuration.
-The algorithm for selecting stripe size takes into account the total number of nodes in the deployment, such that the selected stripe allows for uniform distribution of erasure set drives across all nodes in the pool.
+      The above example deployment consists of 4 nodes with 4 drives each.
+      MinIO initializes with a single erasure set consisting of 16 drives.
 
 Erasure set stripe size dictates the maximum possible :ref:`parity <minio-ec-parity>` of the deployment.
-Use the MinIO `Erasure Coding Calculator <https://min.io/product/erasure-code-calculator>`__ to explore the possible erasure set size and distributions for your planned topology.
-MinIO strongly recommends architecture reviews via |SUBNET| as part of your provisioning and deployment process to ensure long term success and stability.
-As a general guide, plan your topologies to have an even number of nodes and drives where both the nodes and drives have a common denominator of 16.
+   .. figure:: /images/erasure/erasure-coding-possible-parity.svg
+      :figwidth: 100%
+      :align: center
+      :alt: Diagram of possible erasure set parity settings
 
-.. _minio-ec-parity:
+      The above example deployment has an erasure set of 16 drives. 
+      This can support parity between ``EC:0`` and 1/2 the erasure set drives, or ``EC:8``.
+
+   Deployments with a small number of nodes or drives can supported a limited number of parity drives, to a maximum of 1/2 the stripe size.
 
-Erasure Code Parity (``EC:N``)
-------------------------------
+   Erasure Sets have a minimum size of 2 and a maximum size of 16.
```

Comment:
I suggested adding this earlier in the discussion. If you take that suggestion, you can probably drop it here.